workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/utils/getActivationStatusCardLayout.jsx

```diff
@@ -79,7 +79,7 @@ export default function getActivationStatusCardLayout(
 			),
 			id: STATUS_TAG_TYPES.active,
 			subtitle: i18n.translate(
-				'your-liferay-saas-project-is-being-set-up-and-will-be-available-soon'
```

Comment:
@velosofelipe64 we should probably remove the old language key since it is no longer used anywhere

Reply:
Do we still use the In Progress status? Just notice that we use when we have this status, line 111.